documentai/snippets/handle_response_sample_v1beta3.py

```diff
@@ -0,0 +1,215 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# [START documentai_process_summarizer_document]
+# [START documentai_process_custom_extractor_document]
+from typing import Optional
+
+from google.api_core.client_options import ClientOptions
+from google.cloud import documentai_v1beta3 as documentai
+
+
+# TODO(developer): Uncomment these variables before running the sample.
+# project_id = "YOUR_PROJECT_ID"
+# location = "YOUR_PROCESSOR_LOCATION" # Format is "us" or "eu"
+# processor_id = "YOUR_PROCESSOR_ID" # Create processor before running sample
+# processor_version = "rc" # Refer to https://cloud.google.com/document-ai/docs/manage-processor-versions for more information
+# file_path = "/path/to/local/pdf"
+# mime_type = "application/pdf" # Refer to https://cloud.google.com/document-ai/docs/file-types for supported file types
+
+# [END documentai_process_custom_extractor_document]
+
+
+def process_document_summarizer_sample(
+    project_id: str,
+    location: str,
+    processor_id: str,
+    processor_version: str,
+    file_path: str,
+    mime_type: str,
+) -> None:
+    # Optional: Request specific summarization format other than the default
+    # for the processor version.
+    process_options = documentai.ProcessOptions(
+        schema_override=documentai.DocumentSchema(
+            entity_types=[
+                documentai.DocumentSchema.EntityType(
+                    name="summary_document_type",
+                    base_types=["document"],
+                    properties=[
```

Comment:
nit: this is quite hard to read taking into account multiple levels of nesting. Please consider defining property_metadata and properties in a separate variable before earlier in the code for improved readability.

Reply:
Done - It does improve readability, but there's only so much I can do because the API requires a ton of parameters.